ports.mk

```diff
@@ -653,9 +661,10 @@ endif
 	@echo Now run
 	@echo -e "\tgit commit -m 'DISTS: Generated Code::Blocks and MSVC project files'"
 
-
 release-checks:
 	devtools/release-checks.sh
 
 # Mark special targets as phony
-.PHONY: deb bundle osxsnap install uninstall
```

Comment:
I just noticed that the "deb" target was removed from being marked as phony. Was it intended?

Reply:
Yes, as the target doesn't exist anymore in our makefiles.
I mentioned it in commit message.